core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1706,7 +1706,7 @@ public static function addImageToTemplate($template, array $rowData, $maxWidth =
 			->setSize($size)
 			->setLightboxGroupIdentifier($lightboxGroupIdentifier)
 			->setLightboxSize($lightboxSize)
-			->enableLightbox($rowData['fullsize'] ?? false)
+			->enableLightbox($rowData['fullsize'] ?: false)
```

Comment:
Elvis does not check array access, null coalescing does. So this must IMHO stay as it is with an added cast.

```suggestion
			->enableLightbox((bool)($rowData['fullsize'] ?? false))
```